tests/AbstractLoopTest.php

```diff
@@ -16,11 +16,39 @@ public function setUp()
 
     abstract public function createLoop();
 
+    /**
+     * @deprecated Use createSocketPair() instead.
+     */
     public function createStream()
```

Comment:
Removing the method would be a better solution. I just marked it as deprecated to keep it backward compatible.